database/migrations/2016_03_22_124953_rename_phone_number_column_on_groups.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class RenamePhoneNumberColumnOnGroups extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::statement('ALTER TABLE `groups` CHANGE phone_number phone tinyint(1)');
```

Comment:
Is this right?  Phone number now is only a flag?  Where is the phone number actually stored, then?